packages/scoutgame-ui/src/components/common/Hidden.tsx

```diff
@@ -11,21 +11,21 @@ type Props = {
 } & BoxProps;
 
 // TODO: just use class names?
-export function getSXProps({ display = 'block', mdDown, mdUp }: Props) {
+export function getSXProps({ display = 'block', mdDown, mdUp, sx = {} }: Props) {
```

Comment:
Does'nt make sense to pass 'sx' into a function that is just returning the value for 'sx', and its not efficient since u are creating an empty 'sx' prop every time it's called.

I prefer we just change Hidden to do: ```sx={{...getSXProps(), ...sx }}"

Reply:
oops updated